build/ci/runner/Dockerfile.runner

```diff
@@ -1,4 +1,4 @@
-FROM ciscoappnetworking/kubectl_kind:latest
+FROM ciscoappnetworking/kubectl_kind:kubectl.v1.17.5-kind.v0.7.0-helm.v3.2.4
```

Comment:
I prefer an explicit tag that is more descriptive about what is under the image hood, rather than find this information with "docker history"